modules/understanding-openshift-gitops.adoc

```diff
@@ -0,0 +1,39 @@
+[id="understanding-openshift-gitops"]
+= Understanding OpenShift GitOps
```

Comment:
Consider calling it "About..." or "Overview..." "Understanding..." is similar to "Working...." which will be the name of our assembly.